src/Symfony/Component/Validator/Resources/translations/validators.de.xlf

```diff
@@ -322,6 +322,14 @@
                 <source>This is not a valid UUID.</source>
                 <target>Dies ist keine gültige UUID.</target>
             </trans-unit>
+            <trans-unit id="84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
+                <target>Dieser Wert sollte ein Multiplikator von {{ compared_value }} sein.</target>
+            </trans-unit>
+            <trans-unit id="85">
+                <source>This Business Identifier Code (BIC) is not associated with IBAN {{ iban }}.</source>
+                <target>Dieser BIC (internationaler Bankleitzahl) ist nicht mit der IBAN {{ iban }} assoziiert.</target>
```

Comment:
Grammar: "internationale" (without "r" at the end).

Also, I would rather move "BIC" into the brackets: "Diese internationale Bankleitzahl (BIC)…"

Reply:
Articles & conjugation. Still my weak spot after 20 years of speaking German :D